core/src/main/scala/org/apache/spark/ExecutorAllocationManager.scala

```diff
@@ -127,7 +127,7 @@ private[spark] class ExecutorAllocationManager(
   // allocation is only supported for YARN and the default number of cores per executor in YARN is
   // 1, but it might need to be attained differently for different cluster managers
   private val tasksPerExecutorForFullParallelism =
-    conf.getInt("spark.executor.cores", 1) / conf.getInt("spark.task.cpus", 1)
+    conf.get(EXECUTOR_CORES) / conf.getInt("spark.task.cpus", 1)
```

Comment:
Hi, @kiszk . So, this seems to cause many conflicts with your PR because your PR chaning `spark.task.cpus`.
Is it okay for us to proceed @ueshin 's PR first?